engine/context/aci-integration.md

```diff
@@ -0,0 +1,245 @@
+```
+
+### Run a container
+
+Now that you've logged in and created an ACI context, you can start using Docker commands to deploy containers on ACI.
+
+There are two ways to use your new ACI context. You can use the `--context` flag with the Docker command to specify that you would like to run the command using your newly created ACI context.
+
+```console
+docker --context myacicontext run -p 80:80 nginx
+```
+
+Or, you can change context using `docker context use` to select the ACI context to be your focus for running Docker commands. For example, we can use the `docker context use` command to deploy an ngnix container:
+
+```console
+docker context use myacicontext
+docker run -p 80:80 nginx
+```
+
+After you've switched to the `myacicontext` context, you can use docker ps to list your containers running on ACI.
+
+To view logs from your container, run:
+
+```console
+docker logs <CONTAINER_ID>
+```
+
+To execute a command in a running container, run:
+
+```console
+docker exec -t <CONTAINER_ID> COMMAND
+```
+
+To stop and remove a container from ACI, run:
+
+```console
+docker rm <CONTAINER_ID>
+```
+
+## Running Compose applications
+
+You can also deploy and manage multi-container applications defined in Compose files to ACI using the `docker compose` command. To do this:
+
+1. Ensure you are using your ACI context. You can do this either by specifying the `--context myacicontext` flag or by setting the default context using the command  `docker context use myacicontext`.
+
+2. Run `docker compose up` and `docker compose down` to start and then stop a full Compose application.
+
+  By default, `docker compose up` uses the `docker-compose.yaml` file in the current folder. You can specify the working directory using the  --workdir  flag or specify the Compose file directly using the `--file` flag.
+
+  You can also specify a name for the Compose application using the `--project-name` flag during deployment. If no name is specified, a name will be derived from the working directory.
+
+  You can view logs from containers that are part of the Compose application using the command `docker logs <CONTAINER_ID>`. To know the container ID, run `docker ps`.
+
+> **Note**
+>
+> The current Docker Azure integration does not allow fetching a combined log stream from all the containers that make up the Compose application.
+
+## Using ACI resource groups as namespaces
+
+You can create several Docker contexts associated with ACI. Each context must be associated with a unique Azure resource group. This allows you to use Docker contexts as namespaces. You can switch between namespaces using `docker context use <CONTEXT>`.
+
+When you run the `docker ps` command, it only lists containers in your current Docker context. There won’t be any contention in container names or Compose application names between two Docker contexts.
+
+## Install the Docker ACI Integration CLI on Linux

```

Comment:
Maybe we should just point them to the GitHub repo for Linux? That way we don't have to maintain the docs in two places

Reply:
Discussed with Chris.